### PR TITLE
F#359 traj database for shadow ur10

### PR DESCRIPTION
**Pull request by beatrizleon**
Enables the use of different services to plan trajectories based on a set of states previously stored in the database.

### CODE REVIEW

File: project.rosinstall [hunk 1]
```diff
@@ -17,3 +17,13 @@
...
+
+- git:
+    local-name: moveit_ros
+    uri: https://github.com/shadow-robot/moveit_ros.git
+    version: F#276_add_db_state
+
+- git:
+    local-name: moveit_msgs
+    uri: https://github.com/shadow-robot/moveit_msgs.git
+    version: F#276_add_db_state
```
Comment (VahidAminZ) [inline project.rosinstall]: does it complaining about lack of new line at the end of the file?

File: sr_robot_commander/src/sr_robot_commander/follow_warehouse_trajectory.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        self._robot_link = self.group.get_end_effector_link()
+        self._robot_frame = self.group.get_pose_reference_frame()
+
+        self._min_wp_fraction = rospy.get_param("~min_waypoint_fraction", 0.9)
+
+        rospy.sleep(4)
+        rospy.loginfo("Waiting for warehouse services...")
+        rospy.wait_for_service('/moveit_warehouse_services/list_robot_states')
+        rospy.wait_for_service('/moveit_warehouse_services/get_robot_state')
+        rospy.wait_for_service('/moveit_warehouse_services/has_robot_state')
```
Comment (VahidAminZ) [inline sr_robot_commander/src/sr_robot_commander/follow_warehouse_trajectory.py]: remove leading '/'

File: sr_robot_commander/src/sr_robot_commander/follow_warehouse_trajectory.py [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+from moveit_msgs.srv import GetPositionFK
+from sr_robot_msgs.srv import PlanTrajectoryFromList
+from sr_robot_msgs.srv import PlanTrajectoryFromPrefix
+from sr_robot_msgs.srv import ExecutePlannedTrajectory
+
+from std_msgs.msg import Header
+import geometry_msgs.msg
+
+
+class WarehousePlanner:
```
Comment (VahidAminZ) [inline sr_robot_commander/src/sr_robot_commander/follow_warehouse_trajectory.py]: it should be WarehousePlanner(object):

File: sr_robot_msgs/srv/ListNamedTrajectories.srv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+---
+
+string[] names
```
Comment (VahidAminZ) [inline sr_robot_msgs/srv/ListNamedTrajectories.srv]: needs a new line.

**Comment by beatrizleon** [comment]
Fixed #359

**Pull request merged**